src/components/GenericError/style.scss

```diff
@@ -1,23 +1,23 @@
-@import (less) "../Theme/constants.css";
+@import '../Theme/constants';
 
-.icon {
-  height: 144*@unit;
+%icon {
```

Comment:
Minor: should we use the _ for internal only things like this here too?

Reply:
Current configuration does not allow this 🙈 Rule is only set for mixins, as they behave like functions... I'll leave this like this for now, feel free later to adjust.